src/FileSearchApplication.java

```diff
@@ -0,0 +1,55 @@
+/*It is the main class where I am taking the filepath as an input from user and checking the presence of seached word*/
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class FileSearchApplication {
+    protected static String contentOfFile, filepath, searchedWord;
+
+
+    public static void main(String[] args) {
+        try {
+            BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+            System.out.println("enter file path");
+            filepath = reader.readLine();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        System.out.println("Processing....");
+        SpawnThread ch = new SpawnThread();
+        ch.run();
+    }
+
+    /* checking the presence of the searched word and counting its repetation in file and also counting the total no of words in file */
+    public static void isWordExists(String searchedWord) throws SQLException {
+        ArrayList allWordsOfFile = new ArrayList();
+        int repetationOfWord = 0;
+        String temp = "";
+        for (int i = 0; i < contentOfFile.length(); i++) {
+            char eachCharacterOfFile = contentOfFile.charAt(i);
```

Comment:
Check whether you use String tokenizer class or anything similar to it to extract words of each line in the file. You dont have to use rudimentary logic for this

Reply:
Previously I don't have idea about this so now I will use that one insted of all logics.